### ISSUE TITLE
Fetch: test HTTP/1.9 responses?

### ISSUE THREAD

**Issue by annevk** [w3c/web-platform-tests#5064]
@mnot I noticed that Safari doesn't treat responses starting with `HTTP/1.9 ...` as if they started with `HTTP/1.1 ...`. Is that something they should fix?

It's not mentioned at https://wiki.whatwg.org/wiki/HTTP (although that does mention 0.9, which is its own can of worms).

**Comment by mnot** [comment]
In theory, they should accept it as being compatible with the syntax of and downgradable to 1.x. 

In practice, this extension point isn't well-greased; my understanding is that this behaviour is pretty common in servers and other tools.

Do other browsers do it correctly? If they do, it might be worth testing so that it doesn't get any worse, at least.

**Comment by annevk** [comment]
Yeah, other browsers just ignore it. I'll write some tests later today. Guess I'll put them in the http/ directory.
